src/lib.rs

```diff
@@ -214,6 +218,42 @@ impl Encoder {
 		Ok(value)
 	}
 
+	/// Configures the encoder's use of inband forward error correction (FEC).
+	pub fn set_inband_fec(&mut self, value: bool) -> Result<()> {
+		let value = match value {
```

Comment:
This *might* be more succinctly written like `let value = if value { 1 } else { 0 };`;